.github/workflows/build_release.yml

```diff
@@ -0,0 +1,97 @@
+name: Build Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [linux-16.04, macos-latest, windows-latest]
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+
+    - name: install ninja (linux)
```

Comment:
currently we have archives for linux32/64 and win32/64, I assume these are all 64-bit?

Reply:
Yeah.  I don't see any point in supporting lesser used architectures like that here.  People on platforms other than the 3 most common and always build from source right?   What do you think?

Reply:
Seems reasonable to me.